flannel_linux.go

```diff
@@ -135,7 +135,8 @@ func doCmdDel(args *skel.CmdArgs, n *NetConf) (err error) {
 
 	nc := &types.NetConf{}
 	if err = json.Unmarshal(netConfBytes, nc); err != nil {
-		return fmt.Errorf("failed to parse netconf: %v", err)
+		fmt.Fprintf(os.Stderr, "failed to parse netconf: %v", err)
+		return nil
```

Comment:
the consequence is that we are leaving "zombie" interfaces in the bridge, right?

Reply:
Yes, exactly. At least until the node is rebooted.